### PR TITLE
Move to release-it to manage release and CHANGELOG.

### PR DESCRIPTION
**Pull request by lukemelia**
- standard-version has been deprecated

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -88,9 +86,10 @@
     "prettier": "^2.7.1",
     "qunit": "^2.19.1",
     "qunit-dom": "^2.0.0",
+    "release-it": "^14.2.1",
```
Comment (bertdeblock) [inline package.json:89]: I think `create-rwjblue-release-it-setup` creates a slightly outdated setup, we should be able to use v15.

File: package.json [hunk 2]
```diff
@@ -88,9 +86,10 @@
...
+    "release-it-lerna-changelog": "^3.1.0",
```
Comment (bertdeblock) [inline package.json:90]:
```suggestion
    "@release-it-plugins/lerna-changelog": "^5.0.0",
```

The package was renamed, more info in the v5 release:
https://github.com/release-it-plugins/lerna-changelog/releases/tag/v5.0.0

File: package.json [hunk 3]
```diff
@@ -114,8 +120,23 @@
       "ember-named-blocks-polyfill"
     ]
   },
-  "volta": {
-    "node": "14.19.3",
-    "yarn": "1.22.19"
+  "release-it": {
+    "plugins": {
+      "release-it-lerna-changelog": {
```
Comment (bertdeblock) [inline package.json:125]:
```suggestion
      "@release-it-plugins/lerna-changelog": {
```

File: package.json [hunk 4]
```diff
@@ -88,9 +86,10 @@
     "prettier": "^2.7.1",
     "qunit": "^2.19.1",
     "qunit-dom": "^2.0.0",
+    "release-it": "^14.2.1",
+    "release-it-lerna-changelog": "^3.1.0",
     "remarkable": "^2.0.0",
     "sass": "^1.54.8",
-    "standard-version": "^9.5.0",
```
Comment (bertdeblock) [inline package.json:93]: I'm still seeing `standard-version` in the `yarn.lock` file.

**Review by bertdeblock — COMMENTED** [review]
Big fan of this switch 👍

**Comment by lukemelia** [comment]
Thanks for the great catches @bertdeblock

**Pull request merged**